mycore-base/src/main/java/org/mycore/common/MCRSession.java

```diff
@@ -407,7 +407,7 @@ public long getCreateTime() {
      */
     @Deprecated
```

Comment:
Deprecated in 2021.06 - time to delete?

Reply:
Not in this PR. Let's do this together and remove deprecated methods all together in main AFTER we forked release branch